using_images/xpaas_images/eap.adoc

```diff
@@ -121,28 +112,72 @@ If you want the JBoss EAP xPaaS image to use a different JDK version than the de
 
 [NOTE]
-Ensure that you follow the   link:../../creating_images/guidelines.html[guidelines for creating images].
+====
+The variable `*EAP_HOME*` is used to denote the path to the JBoss EAP installation. Replace this variable with the actual path to your JBoss EAP installation.
+====
+
+The S2I process for the xPaaS JBoss EAP image works as follows:
+
+. If there is a *_pom.xml_* file in the source repository, a Maven build is triggered with the contents of `*$MAVEN_ARGS*` environment variable. By default the package goal is used with the OpenShift profile, including the system properties for skipping tests (`*-DskipTests*`) and enabling the Red Hat GA repository (`*-Dcom.redhat.xPaaS.repo.redhatga*`). The results of a successful Maven build are copied to `*EAP_HOME/standalone/deployments*`. This includes all JAR, WAR, and EAR files from the directory within the source repository specified by `*$ARTIFACT_DIR*` environment variable. The default value of `*$ARTIFACT_DIR*` is the target directory.
+
+. Any JAR, WAR, and EAR in the deployments source repository directory are copied to the *_EAP_HOME/standalone/deployments_* directory.
+
+. All files in the configuration source repository directory are copied to *_EAP_HOME/standalone/configuration_*. If you want to use a custom JBoss EAP configuration file, it should be named *_standalone-openshift.xml_*.
+
+. All files in the modules source repository directory are copied to *_EAP_HOME/modules_*.
+
+=== Use a Modified xPaaS JBoss EAP Image
+
+You can make changes to an image or create a custom image to use in OpenShift.
+
+The JBoss EAP configuration file used by OpenShift in the xPaaS JBoss EAP image is *_EAP_HOME/standalone/configuration/standalone-openshift.xml_*. The script to start JBoss EAP is *_EAP_HOME/bin/openshift-launch.sh_*.
 
-== Troubleshooting
+[IMPORTANT]
+====
+Ensure that you have read the link:../creating_images/guidelines.html[guidelines for creating images] and follow them when creating a modified image.
+====
+
+To use a modified image in OpenShift:
+
+[WARNING]
+====
+This procedure results in losing configuration placeholders for various settings such as datasources, messaging, HTTPS, KeyCloak, etc. A workaround for this issue is to create a duplicate copy of the *_standalone.xml_* file to edit. The original and edited versions can be compared after all edits are complete and placeholder values can be copied to the edited version from the original version to retain these values.
+====
+
+. Run the xPaaS JBoss EAP image using Docker.
+
+. Make the required changes using the JBoss EAP Management CLI by running the script at *_EAP_HOME/bin/jboss-cli.sh_*.
+
+. Commit the changed container as a new image and then use the modified image in OpenShift.
+
+=== Troubleshooting
+
+If an application is not starting, use the following command to view details to locate and troubleshoot the problem:
+
+----
+$ oc describe po {pod_name}
```

Comment:
if `{pod_name}` is a variable, please use angle brackets instead of curly braces.